src/room/GroupCallView.tsx

```diff
@@ -90,16 +89,39 @@ export function GroupCallView({
   }, [groupCall]);
 
   const { displayName, avatarUrl } = useProfile(client);
+  const roomName = useRoomName(groupCall.room);
+  const roomAvatar = useRoomAvatar(groupCall.room);
+  const roomEncrypted = useIsRoomE2EE(groupCall.room.roomId)!;
+
   const matrixInfo = useMemo((): MatrixInfo => {
     return {
       userId: client.getUserId()!,
       displayName: displayName!,
       avatarUrl: avatarUrl!,
       roomId: groupCall.room.roomId,
-      roomName: groupCall.room.name,
+      roomName,
       roomAlias: groupCall.room.getCanonicalAlias(),
+      roomAvatar,
+      roomEncrypted,
     };
-  }, [client, displayName, avatarUrl, groupCall]);
+  }, [
+    displayName,
+    avatarUrl,
+    groupCall,
+    roomName,
+    roomAvatar,
+    roomEncrypted,
+    client,
+  ]);
+
+  const participatingMembers = useMemo(() => {
+    const members: RoomMember[] = [];
+    for (const [member, deviceMap] of participants.entries()) {
+      // Repeat the member for as many devices as they're using
+      for (let i = 0; i < deviceMap.size; i++) members.push(member);
```

Comment:
Is this according to the designs? It feels a bit weird to me

Reply:
The designs don't say anything about what should count as a participant, so this was just my doing. But good point, now that you mention it, counting multiple devices as a single participant sounds less surprising to me. I would note that also Element Web is currently doing it this way (repeating the member if they're on multiple devices), so we should change EW to be consistent eventually.